src/Altinn.Platform/Altinn.Platform.Authorization/IntegrationTests/Models/DataElement.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
```

Comment:
just wondering if we can't reuse the models from Storage.Interface.Models

Reply:
Fixed it now. Removed models and just imported the nuget